src/document/Document.js

```diff
@@ -196,12 +201,15 @@ define(function (require, exports, module) {
      */
     Document.prototype._makeEditable = function (masterEditor) {
         if (this._masterEditor) {
-            console.error("Document is already editable");
-        } else {
-            this._text = null;
-            this._masterEditor = masterEditor;
-            masterEditor.on("change", this._handleEditorChange.bind(this));
+            //Already a master editor is associated , so preserve the old editor in list of full editors
+            if(this._associatedFullEditors.indexOf(this._masterEditor) < 0){
```

Comment:
You need to use Brackets with JSLint enabled when contributing to Brackets. Please re-check your code. In this specific case, there's a space missing between `if` and `(` which JSLint would complain about.